docs/source/tutorials.md

```diff
@@ -0,0 +1,47 @@
+# Tutorials
+
+On this page we provide step-by-step video tutorials on how to use the AMDirT graphical-user-interface (GUI) of the `filter` command.
+
+A [full walkthrough](#full-walkthrough) can be seen at the end of this page.
+
+## Loading AMDirT filter
+
+This tutorial shows how to activate a [pre-created](README#install) conda environment from a command-line interface, and load the graphical-user interface (GUI) in your browser.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/EANpFh__6V8" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
```

Comment:
This step is only needed because AMDirT is not yet released. But loading a conda environment won't be necessary once it's released on pip/bioconda.